src/base.ts

```diff
@@ -80,6 +82,7 @@ type Method = "GET" | "POST" | "PUT" | "PATCH" | "DELETE"
 
 interface RequestOptions extends AuthorizedApiCallOptions {
   method?: Method
+  apiVersion?: ApiVersion
```

Comment:
Don't you need to add it to the `ApiCallOptions` Interface so it's available/typed for external and POST/PUT/DELETE requests as well?

Reply:
indeed, just noticed that as well when working on the actual implementation

Reply:
It's already there as you extend `AuthorizedApiCallOptions` also this is the internal interface so we probably wanna add it to the base interface `ApiCallOptions` instead...

Reply:
the applied change seems to do the trick

Reply:
oke might have messed up something then :sli